setup/nextcloud.sh

```diff
@@ -285,47 +302,48 @@ if [ \( $? -ne 0 \) -a \( $? -ne 3 \) ]; then exit 1; fi
 
 # Set PHP FPM values to support large file uploads
 # (semicolon is the comment character in this file, hashes produce deprecation warnings)
-tools/editconf.py /etc/php/7.2/fpm/php.ini -c ';' \
-	upload_max_filesize=16G \
-	post_max_size=16G \
-	output_buffering=16384 \
-	memory_limit=512M \
-	max_execution_time=600 \
-	short_open_tag=On
+../tools/editconf.py /etc/php/7.2/fpm/php.ini -c ';' \
```

Comment:
This breaks when running the script from the mailinabox directory

Reply:
whoops, I was running from setup dir.